saos-persistence/src/main/java/pl/edu/icm/saos/persistence/model/Judgment.java

```diff
@@ -573,8 +576,16 @@ public void passVisitorDown(Visitor visitor) {
         }
     }
 
-    @PreUpdate
-    protected void onUpdate(){
+    /**
+     * Updates modification date to current time.
```

Comment:
Sets the modification date to the current time.
<br/>
This method is used in persistence repository classes, so that saving the object updates the modification date. You should not use it in other classes.